Tools/autotest/param_metadata/mdemit.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+"""
+Emit parameter documentation in markdown format
+"""
+
+from param import known_param_fields
+from emit import Emit
+import re
+
+# Parameter groups disabled at compile time
+sub_blacklist = ['AVOID_', 'CIRCLE_', 'FLOW', 'MIS_', 'PRX', 'RALLY_', 'RCMAP_', 'RPM', 'TERRAIN_', 'WPNAV_']
+
+# Parameter groups with redundant information (ie RCn_, SERVOn_)
+nparams = ['RCn_', 'SERVOn_', 'SRn_', 'BTNn_']
+
+class MDEmit(Emit):
+
+    def __init__(self):
+        Emit.__init__(self)
+        fname = 'parameters.md'
+        self.nparams = []
+        self.f = open(fname, mode='w')
+        self.header = """---
```

Comment:
You're right, this is for our [ardusub-docs](https://github.com/bluerobotics/ardusub-docs) navigation. I can take it out, or make it an option (off by default).